scripts/generate_dump

```diff
@@ -642,6 +892,40 @@ get_asic_count() {
     echo `eval ${cmd} 2>&1`
 }
 
+###############################################################################
+# Get counter snapshot
+# Globals:
+#  None
+# Arguments:
+#  asic_name: Name of the asic vendor
+#  index: Index of counter snapshot
+# Returns:
+#  None
+###############################################################################
+save_counter_snapshot() {
+    local asic_name="$1"
+    local idx=$2
+    counter_t=$(date +'%d/%m/%Y %H:%M:%S:%6N')
+
+    save_cmd "echo $counter_t" "date.counter_$idx"
+    save_cmd "show interface counters" "interface.counters_$idx"
+    save_cmd "show queue counters" "queue.counters_$idx"
+    save_redis "COUNTERS_DB" "COUNTERS_DB_$idx"
+
+    if [ "$asic_name" = "broadcom" ]; then
+        save_cmd "cat /proc/bcm/knet/dstats" "broadcom.knet_drop.counters_$idx"
+        save_cmd "cat /proc/bcm/knet/stats" "broadcom.knet_filter.counters_$idx"
+        if [ -e /usr/local/bin/softnet_stat.sh ]; then
+            save_cmd "softnet_stat.sh" "softnet_queue.counters_$idx"
+        fi
+        if [ -e /proc/bcm/knet/rx_drop ]; then
+            save_cmd "cat /proc/bcm/knet/rx_drop" "broadcom.knet_queue.counters_$idx"
+        fi
+    fi
+    save_cmd "netstat -i" "netstat.counters_$idx"
+    save_cmd "ifconfig -a" "ifconfig.counters_$idx"
```

Comment:
use "save_cmd_all_ns" for these two commands

Reply:
Done